src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -132,7 +132,7 @@
     <DefineConstants Condition="'$(FeatureManagedEtwChannels)' == 'true'">$(DefineConstants);FEATURE_MANAGED_ETW_CHANNELS</DefineConstants>
     <DefineConstants Condition="'$(FeaturePerfTracing)' == 'true'">$(DefineConstants);FEATURE_PERFTRACING</DefineConstants>
     <DefineConstants Condition="'$(FeatureDefaultInterfaces)' == 'true'">$(DefineConstants);FEATURE_DEFAULT_INTERFACES</DefineConstants>
-    <DefineConstants Condition="'$(TargetsUnix)' == 'true'">$(DefineConstants);TARGET_UNIX</DefineConstants>
+    <DefineConstants Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'">$(DefineConstants);TARGET_UNIX</DefineConstants>
```

Comment:
😢  I don't think we should do this or at least if we do we should have an issue tracking removing it.

Reply:
Where is TARGET_UNIX consumed? I only see 3 files (all in libraries\System.Private.CoreLib\src\System\Threading). Wouldn't it be easier to just change those to `#if TARGET_UNIX | TARGET_BROWSER` if that's what you need?